### ISSUE TITLE
cwrappers for inequality

### ISSUE THREAD

**Issue by aqui18** [symengine/symengine#1622]
Hello,

I'm looking to implement inequality operators such as the ones found in [here](https://github.com/symengine/symengine/blob/master/symengine/logic.h#L250) for my wrapper library.
eg. `a < b < 1`

However, it seems that they have not been implemented yet in `cwrappers.cpp`.

Is this correct?
Or is there a different way to wrap these operators.

**Comment by isuruf** [comment]
Yes, they haven't been wrapped in C yet. PRs are welcome.

**Comment by aqui18** [comment]
Great thank you.
I'll see if I can get a PR going.

**Comment by aqui18** [comment]
Are functions such as `div` in the expanded form because of historical reasons?
ie.
```cpp
CWRAPPER_OUTPUT_TYPE basic_div(basic s, const basic a, const basic b)
{
    CWRAPPER_BEGIN
    s->m = SymEngine::div(a->m, b->m);
    CWRAPPER_END
}
```
rather than
`IMPLEMENT_TWO_ARG_FUNC(div)`

So, going forward you would rather the macro format if possible?

**Comment by isuruf** [comment]
Yes, that implementation predates the macro `IMPLEMENT_TWO_ARG_FUNC` and yes, we'd prefer using the macro.
